src/array_api_stubs/_draft/manipulation_functions.py

```diff
@@ -199,6 +199,24 @@ def stack(arrays: Union[Tuple[array, ...], List[array]], /, *, axis: int = 0) ->
     """
 
 
+def unstack(array: array, /, *, axis: int = 0) -> Tuple[array]:
```

Comment:
The parameter name should be called `x` like in pretty much all other functions.

Reply:
The type annotation should be `Tuple[array, ...]` to indicate a variable number of tuple elements.

Reply:
@steff456 you made the wrong change here - the comment is about the return type, not the first input parameter.